frontend/src/hooks/useMaxMultiplier.js

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+
+const backendUrl = process.env.REACT_APP_BACKEND_URL || 'http://0.0.0.0:8000';
+
+export const useMaxMultiplier = () => {
+  const { data, isLoading, error } = useQuery({
```

Comment:
please use latest version of `TanStack Query` (v5), in which `isLoading` doesn't exist, instead it was renamed  to `IsPending`